indexer/provider.go

```diff
@@ -34,6 +37,22 @@ func (a *IndexerApp) handleModProviderEvent(evt types.ModProviderEvent) error {
 	if _, err = a.db.InsertModProviderEvent(provider.ID, evt); err != nil {
 		return errors.Wrapf(err, "error inserting ModProviderEvent for %s chain %s", evt.Pubkey, evt.Chain)
 	}
+
+	if isMetaDataUpdated {
```

Comment:
I generally favor using the inverse comparison and return early.
```
if !isMetaDataUpdated {
   return nil
}
```

Reply:
check, let me fix that